src/components/Footer.astro

```diff
@@ -113,8 +113,8 @@ const groups: LinkGroup[] = [
 	<div
 		class="grid grid-cols-[1fr,auto] items-center gap-x-4 gap-y-16 py-8 opacity-60 md:grid-cols-[auto,1fr,auto] md:py-12"
 	>
-		<a class="link" href="/privacy">Privacy Policy</a>
-		<a class="link mr-auto" href="/terms">Terms of Service</a>
+		<a class="link" href="/privacy" rel="prefetch">Privacy Policy</a>
+		<a class="link mr-auto" href="/terms" rel="prefetch">Terms of Service</a>
```

Comment:
Lets skip these two for now too, they're probably so rarely clicked it's not worth the extra network requests when a user scrolls to the end